packages/popup/src/pages/CurrentSeed/index.js

```diff
@@ -66,12 +66,15 @@ function CurrentSeed() {
   const {setLoading} = useLoading()
 
   useEffect(() => {
-    setAccountNamePlaceholder(`Seed-1-${hdGroup[0]?.account?.length + 1}`)
+    setAccountNamePlaceholder(
+      `${t('seedPrefix')}1-${hdGroup[0]?.account?.length + 1}`,
+    )
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [hdGroup])
   const onClickGroup = index => {
     setSelectedGroupIdx(index)
     setAccountNamePlaceholder(
-      `Seed-${index + 1}-${hdGroup[index].account.length + 1}`,
+      `${t('seedPrefix')}${index + 1}-${hdGroup[index].account.length + 1}`,
```

Comment:
create account name with hardcode not i18n

Reply:
done